commands/init.go

```diff
@@ -61,6 +61,10 @@ func InitializeDirectory(ctx *cli.Context) error {
 	_ = initConfigGroup(configs.PrysmValidatorConfigDependencies)
 	log.Info("✅  Prysm validator configuration files downloaded!\n\n")
 
+	log.Info("⬇️  Downloading teku configuration files...")
+	_ = initConfigGroup(configs.TekuConfigDependencies)
+	log.Info("✅  Prysm validator configuration files downloaded!\n\n")
```

Comment:
should it be `Teku validator configuration files downloaded!` ?

Reply:
Yes, you are right, adjusted :+1: